denizen_scripts/survival/repo-link/spawn_scripts/player_joins_server.dsc

```diff
@@ -8,7 +8,7 @@ player_joins_server:
         - teleport spawn
         - flag player first_joined:!
         - wait 2s
-        - narrate "<&2>- - - - - - - - - - - - - - - - - - - -<&nl><&nl>       <&6>Welcome to <&2><&l>Adriftus MC<&r><&6> <player.display_name>!<&nl><&nl><&2>- - - - - - - - - - - - - - - - - - - -"
+        - narrate "<&2>- - - - - - - - - - - - - - - - - - - -<&nl><&nl>       <&6>Welcome to <&2><&l>Adriftus MC<&r><&6> <player.display_name>!<&nl><&nl><&2>- - - - - - - - - - - - - - - - - - - -<&nl><&a>Use <&6>/play<&a> to join another server."
```

Comment:
this was already added as an additional message [Here](https://github.com/Adriftus-Studios/network-script-data/blob/ac420ebfa95c7d9fd0f98758dd4e3befb727dd26/denizen_scripts/survival/repo-link/temporary/player_join_messages.dsc#L7)
```yml
- narrate "<&4>[<&c>Note<&4>] <&b>This server is in <&6>ALPHA<&3>. <&b>Until released, you can play semi-survival on <&6>BehrCraft<&b> by typing <&3><&dq><&6>/<&e>play behrcraft<&3><&dq>"
```
It also notates the only currently live server (Xeane's still appears here), as well as gives information about being an alpha server. We've had a few complaints when people started playing before realizing elsewise

Reply:
```suggestion
        - narrate "<&2>- - - - - - - - - - - - - - - - - - - -<&nl><&nl>       <&6>Welcome to <&2><&l>Adriftus MC<&r><&6> <player.display_name>!<&nl><&nl><&2>- - - - - - - - - - - - - - - - - - - -"
```